site/lib/src/pages/custom_pages.dart

```diff
@@ -66,3 +68,32 @@ MemoryPage get _devtoolsReleasesIndex => MemoryPage.builder(
     return const Component.empty();
   },
 );
+
+MemoryPage get _fweTestingPage => const MemoryPage(
+  path: 'fwe.md',
+  content: '''
+---
+title: FWE Testing Page
+description: This is a test page for experimenting with First Week Experience (FWE) features.
+sitemap: false
+---
+
+<Quiz>
+- question: What is the Effective Dart guideline for the first sentence of a documentation comment?
```

Comment:
When you have a chance, can you add at least one more question to this testing quiz? Thanks!